edge-apps/rss-reader/screenly.yml

```diff
@@ -0,0 +1,34 @@
+app_id: 01H5X26SAC194MEJDFSMV7BFTD
+root_asset_id: 01H5X2X3VHA3AJD84TZSHD4ZV9
+user_version: v1.0.0
+revision: 1
+description: Screenly RSS Reader App
+icon: ''
+author: Screenly, Inc.
+homepage_url: ''
+settings:
+  disable_analytics:
```

Comment:
Add setting for RSS feed URL and title for RSS feed.

Reply:
Got it, thanks. I've already added a setting for RSS feed URL in this commit. Stay tuned for the other settings.